components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-local.jsp

```diff
@@ -575,12 +604,223 @@ action="idp-mgt-get-ridp-metadata-finish-ajaxprocessor.jsp"></form>
+                <a href="#"><%=Encode.forHtmlAttribute(connectorConfig.getFriendlyName())%>
+                </a>
+            </h2>
+            <div class="toggle_container sectionSub" style="margin-bottom:10px; display: none;" id="roleConfig2">
+                <table class="carbonFormTable">
+                    <%
+                    org.wso2.carbon.identity.governance.stub.bean.Property[] connectorProperties = connectorConfig.getProperties();
+                        for (int k = 0; k < connectorProperties.length; k++) {
+                            String value = connectorProperties[k].getValue();%>
+                        <tr>
+                            <td style="width: 500px;">
+                                <%=Encode.forHtmlAttribute(connectorProperties[k].getDisplayName())%>
+                            </td>
+                            <%
+                                if (value.equalsIgnoreCase("true") || value.equalsIgnoreCase("false")) {
+                                    // assume as boolean value. But actually this must be sent from backend.
+                                    // will fix for next release.
+                            %>
+                            <td>
+                                <input class="sectionCheckbox" type="checkbox"
+                                       onclick="setBooleanValueToTextBox(this)"
+                                        <%if (Boolean.parseBoolean(value)) {%> checked="checked" <%}%>
+                                       value="<%=Encode.forHtmlAttribute(connectorProperties[k].getName())%>"/>
+                            <%
+                            if (StringUtils.isNotBlank(connectorProperties[k].getDescription())) {%>
+                                <div class="sectionHelp">
+                                       <%=Encode.forHtmlAttribute(connectorProperties[k].getDescription())%>
+                                </div>
+                            <%}%>
+                                <input id="<%=Encode.forHtmlAttribute(connectorProperties[k].getName())%>"
+                                       name="<%=Encode.forHtmlAttribute(connectorProperties[k].getName())%>"
+                                       type="hidden" value="<%=Encode.forHtmlAttribute(value)%>"/>
+                            </td>
+                            <%
+                            } else {%>
+                            <td colspan="2"><input type="text" name=<%=Encode.forHtmlAttribute(connectorProperties[k].getName())%>
+                                                   id=<%=Encode.forHtmlAttribute(connectorProperties[k].getName())%>
+                                                   style="width:400px"
+                                                   value="<%=Encode.forHtmlAttribute(value)%>"/>
+                            <%
+                            if (StringUtils.isNotBlank(connectorProperties[k].getDescription())) {%>
```

Comment:
If we do so it will complex the code drawing logic and reduce the readability of the code